specification/monitor/resource-manager/Microsoft.Insights/preview/2020-05-01-preview/scheduledQueryRule_API.json

```diff
@@ -405,6 +414,16 @@
         "properties"
       ],
       "properties": {
+        "kind" :{
```

Comment:
This is a fix to s360 violation :
https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=C09C7410-CA54-4450-96BF-BF46F64F2190